packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -91,6 +93,14 @@ contract Deploy is Deployer {
         setCannonFaultGameImplementation();
 
         transferDisputeGameFactoryOwnership();
+
+        ChainAssertions.postDeployAssertions(
```

Comment:
Now the assertions only run when the `run()` entrypoint is used but there are no assertions ran when other entrypoints are used. Ideally the assertions are placed in the same functions that previously had assertions so that we can still have the same sort of assertions run no matter what entrypoint is used

Reply:
Good point. I wasn't aware that the `Deployer` lets you select a different entrypoint. Will make this change.

Reply:
It's done.